dll/directx/wine/ddraw/ddraw.c

```diff
@@ -1145,6 +1146,10 @@ static HRESULT WINAPI ddraw7_SetDisplayMode(IDirectDraw7 *iface, DWORD width, DW
                 ddrawformat_from_wined3dformat(&ddraw->primary->surface_desc.u4.ddpfPixelFormat, mode.format_id);
         }
         ddraw->flags |= DDRAW_RESTORE_MODE;
+		
+		if (ddraw->cooperative_level & DDSCL_EXCLUSIVE)
```

Comment:
Please address all comments in https://github.com/reactos/reactos/pull/4724/files#r979187024
Creating a new PR, blatantly disregarding reviewer comments is quite rude.

Reply:
Hi! Sorry I mistakenly merged the branch with my forked repository which created to duplicated if-blocks and I pushed the branch without reverting the merge, which made more of a mess. So I had to delete that branch because although I hard pushed to revert the changes, duplicate if-blocks were still showing up. Which would have messed up the codes. My mistake sorry.

Reply:
On that note, the review you linked to is out of date because it is using code from a WINE commit.